marklogic-data-hub-central/ui-custom/e2e/cypress/integration/searchValidations.spec.tsx

```diff
@@ -0,0 +1,132 @@
+import landingPage from "../support/pages/landing";
+import searchPage from "../support/pages/search";
+
+var searchPerson="";
+var searchAddress="";
+var searchPhone="";
+var searchEmail="";
+var searchSources="";
+var searchStatus="";
+var searchCountries="";
+
+describe("Search Validations ", () => {
+	beforeEach(() => {
+    cy.intercept("/api/explore**").as("getSearch");
+  });
+  it("Visit Entity Viewer Application ", () => {
+    cy.visit("/");
+    cy.waitUntil(() => landingPage.whatsNewChart().should("be.visible",{timeout:5000}));
+  });
+  it("Click on serach button for navigating to search page and get search values", () => {
+    searchPage.searchButton().click();
+    cy.waitUntil(() => searchPage.resultsList().should("be.visible",{timeout:5000}));
```

Comment:
Can we remove the wait untils or are we using it for some reason? We actually don't need them, the should() commands waits already :)